app/facade/entreprise_with_attestations_facade.rb

```diff
@@ -7,7 +7,7 @@ class EntrepriseWithAttestationsFacade
 
   def initialize(token:, siren:)
     @token = token
-    @siren = siren.try(:strip)
+    @siren = siren.try(:split).try(:join)
```

Comment:
c'est tordu un peu 😅

```suggestion
    @siren = (siren || '').gsub(/\s+/, '')
```